src/SDKs/DataFactory/Management.DataFactory/Generated/Models/AzureStorageLinkedService.cs

```diff
@@ -86,7 +86,7 @@ public AzureStorageLinkedService(IDictionary<string, object> additionalPropertie
         /// manager. Type: string (or Expression with resultType string).
         /// </summary>
         [JsonProperty(PropertyName = "typeProperties.encryptedCredential")]
-        public object EncryptedCredential { get; set; }
+        public string EncryptedCredential { get; set; }
```

Comment:
@ligengatms Since there are breaking changes being introduced here, the SDK version must be updated [here](https://github.com/Azure/azure-sdk-for-net/blob/51ada63b251742843022e51bbcd8e1def8128d56/src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj) to `3.0.0`

Reply:
Hi @dsgouda this is not a breaking change actually, this field is not an input field for customer (customer could not provide it by create or update operation ), it's only in response (for get operation), and this field will always be a string value, that's why we do this correction in the meantime.

Reply:
@shahabhijeet can you take a look?

Reply:
Thanks for the explanation. Couple of questions here:
- What the operations where this model (AzureStorageLinkedService) is returned by the server?
- If `EncryptedCredential` is supposed to be set by the service only, it should have been marked as `readonly` in the REST spec. Please reassess the spec and mark all the readonly properties appropriately.

Reply:
Thanks for pointing it out.
We support CRUD operations for AzureStorageLinkedService, and it's returned by LinkedServiceOperation.GetWithHttpMessagesAsync (If I understand your question correctly).
For changing all the readonly properties for REST spec, it requires some efforts to review and change the whole DataFactory spec, could we have this PR merged , and on the other hand I would do the effort of marking all the readonly properties?
Thanks for your time.

Reply:
Thanks for the info.
We won't block this PR for now but please create a work item at your end to update the REST specs for correctness. Also, please note that once the properties are marked as readonly and SDK is regenerated, it becomes a breaking change to the end users.

Reply:
Since there are other non-breaking additive changes, the version must be updated to `2.1.0`
Please update the `PackageReleaseNotes` and `AssemblyInfo.cs` accordingly

Reply:
Thanks for the detailed instructions. I'm talking with our rest API owner Hermine with correcting the readonly field. and for updating the sdk version, updated, thanks again for your help.